### ISSUE TITLE
Search should scroll window to show object

### ISSUE THREAD

**Issue by ghost** [pure-data/pure-data#403]
How to reproduce:

- Open Pd and go to **Help** -> **List of objects...**
- Hit `Ctrl` + `f` and enter a search term, e.g. "delay" and press `Enter`
- The search dialog will now say this:
> Showing '1' out of 48 items in delay
- The viewing region of the patch remains at the top, so there's no way to know *where* in the patch this result is.

It would be better for the viewing region to jump down (or up, or sideways) to the search result, as this is how search works in most other programs (text editors, web browsers, etc).

**Comment by umlaeute** [comment]
btw, with Pd-0.48-1 the `[delay]` object seems to be in the visible part of the canvas anyhow (the canvas being 733 pixes high and `[delay]` residing on y=660).

but of course, this doesn't change a ι of the original problem...

**Comment by hjek** [comment]
> the canvas being 733 pixes high and `[delay]` residing on y=660).

Yes, forgot that not everyone are using 1024x600 displays today :-)
